src/tlo/methods/rti.py

```diff
@@ -3845,27 +3954,28 @@ def apply(self, person_id, squeeze_factor):
                 get_item_code('Blood, one unit'): 2,
                 get_item_code("Oxygen, 1000 liters, primarily with oxygen cylinders"): 23_040
             }
-            is_cons_available = self.get_consumables(
-                self.module.item_codes_for_consumables_required['shock_treatment_child']
-            )
+            # cons_available = self.get_consumables(item_codes=self.module.cons_item_codes['shock_treatment_child'])
+            cons_available = self.get_consumables(item_codes=
+                  self.module.item_codes_for_consumables_required['shock_treatment_child'])
         else:
             self.module.item_codes_for_consumables_required['shock_treatment_adult'] = {
                 get_item_code("ringer's lactate (Hartmann's solution), 1000 ml_12_IDA"): 2000,
                 get_item_code('Cannula iv  (winged with injection pot) 18_each_CMST'): 1,
                 get_item_code('Blood, one unit'): 2,
                 get_item_code("Oxygen, 1000 liters, primarily with oxygen cylinders"): 23_040
             }
-            is_cons_available = self.get_consumables(
-                self.module.item_codes_for_consumables_required['shock_treatment_adult']
-            )
+            # cons_available = self.get_consumables(item_codes=self.module.cons_item_codes['shock_treatment_adult'])
+            cons_available = self.get_consumables(item_codes=
+                  self.module.item_codes_for_consumables_required['shock_treatment_adult'])
```

Comment:
same comment as above about renaming of variables. (i.e. suggest to roll it back -- unless you feel very strongly?)